app/controllers/search_controller.rb

```diff
@@ -41,8 +41,7 @@ def fill_in_slimmer_headers(result_count)
     set_slimmer_headers(
       result_count: result_count,
       format:       "search",
-      section:      "search",
-      proposition:  "citizen"
+      section:      "search"
```

Comment:
A point of moderate pedantry: it looks like this code [dates back](https://github.com/alphagov/frontend/commit/68d79169a77669e19d92b5f68b66c683d05ffb71) to before we [added trailing commas to our style guide](https://github.com/alphagov/styleguides/commit/4da34b1848772ee06084ef3204601f34615f85d6), so is it worth changing the style while we’re here and keeping the trailing comma after the last line to give us nicer diffs?